From a6461f6e486b61bc4e038d53d42120795d525b61 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.4-a6461f6e486-20211015T123619Z-avarab@gmail.com>
In-Reply-To: <cover-0.4-00000000000-20211015T123619Z-avarab@gmail.com>
References: <cover-0.4-00000000000-20211015T123619Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 15 Oct 2021 13:00:31 +0200
Subject: [PATCH 1/4] doc lint: fix error-hiding regression
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the broken "make lint-docs" (or "make check-docs" at the
top-level) target, which has been broken since my cafd9828e89 (doc
lint: lint and fix missing "GIT" end sections, 2021-04-09).

The CI for "seen" is emitting an error about a broken gitlink, but due
to there being 3x scripts chained via ";" instead of "&&" we're not
carrying forward the non-zero exit code.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index f5605b7767f..78324934d9f 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -461,9 +461,9 @@ lint-docs::
 		$(HOWTO_TXT) $(DOC_DEP_TXT) \
 		--section=1 $(MAN1_TXT) \
 		--section=5 $(MAN5_TXT) \
-		--section=7 $(MAN7_TXT); \
-	$(PERL_PATH) lint-man-end-blurb.perl $(MAN_TXT); \
-	$(PERL_PATH) lint-man-section-order.perl $(MAN_TXT);
+		--section=7 $(MAN7_TXT) && \
+	$(PERL_PATH) lint-man-end-blurb.perl $(MAN_TXT) && \
+	$(PERL_PATH) lint-man-section-order.perl $(MAN_TXT)
 
 ifeq ($(wildcard po/Makefile),po/Makefile)
 doc-l10n install-l10n::
-- 
2.33.1.1338.g20da966911a

